qt/scientific_interfaces/Muon/MuonAnalysisHelper.cpp

```diff
@@ -173,10 +173,12 @@ void printRunInfo(MatrixWorkspace_sptr runWs, std::ostringstream &out) {
   out << "\nAverage Temperature: ";
   if (run.hasProperty("Temp_Sample")) {
     // Filter the temperatures by the start and end times for the run.
-    run.getProperty("Temp_Sample")->filterByTime(start, end);
-
-    // Get average of the values
-    double average = run.getPropertyAsSingleValue("Temp_Sample");
+    Mantid::Kernel::SplittingInterval time_split(start, end);
+    std::vector<Mantid::Kernel::SplittingInterval> splitVector = {time_split};
+    auto Temp_Sample = run.getProperty("Temp_Sample");
+    const auto *Temp_Sample_TimeSeries =
```

Comment:
A couple of things:
  - variables names should be camelCase
  - we should check the result of the `dynamic_cast` for safety or we'll hit a segfault